givitsite/friendreq/tests.py

```diff
@@ -1,3 +1,45 @@
-from django.test import TestCase
+from django.test import TestCase, RequestFactory
+from django.shortcuts import reverse
+from django.contrib.auth.models import User
+from .models import ItemRequest,ItemsFound
+from django.conf import settings
 
-# Create your tests here.
+
+
+url_list = ['/friend/request','/friend/feed']
+url_name_list = ['requestItem','itemRequest_create_view']
+
+class FriendPageTests(TestCase):
+
+    # check the status code when navigate to the given url
+    def test_request_status_code(self):
+        for url in url_list:
+            response = self.client.get(url)
+            self.assertEquals(response.status_code,200)
```

Comment:
Have you used flake8 on this code?

Reply:
@lioramilbaum so we want to use flake8 in our IDE or just in the CI process?

Reply:
fixed!
I used flake8